Makefile

```diff
@@ -1,24 +1,23 @@
 .PHONY: requirements
 
 requirements: requirements.js
-	pip install -q -r requirements/base.txt --exists-action w
+	pip install .
 
 requirements.js:
 	npm install
 
-requirements.test: requirements
-	pip install -q -r requirements/test.txt --exists-action w
-
 install: requirements
 
-develop: requirements.test
+develop: requirements
+	pip install -U pytest>=2.7 pytest-mock pycodestyle pylint edx-lint==0.5.1
```

Comment:
I'd prefer these be in a requirements file. Updates are easier to follow over time if it's in a file with line breaks (for example)